src/main/java/com/hyperwallet/clientsdk/util/HyperwalletMultipartUtils.java

```diff
@@ -0,0 +1,71 @@
+package com.hyperwallet.clientsdk.util;
+
+import com.hyperwallet.clientsdk.HyperwalletException;
+import com.hyperwallet.clientsdk.model.HyperwalletVerificationDocument;
+import net.minidev.json.JSONObject;
+import org.apache.commons.lang3.StringUtils;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class HyperwalletMultipartUtils {
+
+    public static Multipart convert(List<HyperwalletVerificationDocument> uploadList) throws IOException {
+
+        JSONObject document = new JSONObject();
+        Multipart multipartList = new Multipart();
+        for (HyperwalletVerificationDocument uploadData : uploadList) {
+
+            addDocumentValue(document, "type", uploadData.getType());
+            if (!StringUtils.isEmpty(uploadData.getCountry())) document.put("country", uploadData.getCountry());
+            if (!StringUtils.isEmpty(uploadData.getCategory())) document.put("category", uploadData.getCategory());
+            if (!StringUtils.isEmpty(uploadData.getStatus())) document.put("status", uploadData.getStatus());
```

Comment:
You missed these changes:

```suggestion
            addDocumentValue(document, "country", uploadData.getCountry());
            addDocumentValue(document, "category", uploadData.getCategory());
            addDocumentValue(document, "status", uploadData.getStatus());
```